Modules/SubModule_Import/xASL_imp_NII2BIDS.m

```diff
@@ -63,14 +61,13 @@ function xASL_imp_NII2BIDS(imPar, studyPath, studyParPath)
 	
 	spm_jsonwrite(fullfile(imPar.BidsRoot,[bidsPar.datasetDescription.filename '.json']),datasetDescription);
 	
-	%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 	%% 3. Go through all subjects and check all the M0 and ASLs and modify the JSONs
 	% This step should be completely automatic, just taking the info filled above and using it to convert to full BIDS.
 	
 	% Go through all subjects
-	listSubjects = xASL_adm_GetFileList(imPar.TempRoot,[],false,[],true);
-    for iSubject = 1:length(listSubjects)
-        xASL_imp_NII2BIDS_Subject(imPar,bidsPar,studyPar,listSubjects{iSubject});
+	listSubjectsSessions = xASL_adm_GetFileList(imPar.TempRoot,[],false,[],true);
```

Comment:
I would add a few folder names of which we're sure that they shouldn't be subjects, like `population`, but we can do that later. Also, the folder name should have at least one letter or at least one digit. Now, folders named like `*&` or `temp` can become a subject as well

Reply:
Yeah, we can do things like these. I didn't change this line though, I only moved it. Maybe keep it like it is for now and then optimize later on?